auth/debug.go

```diff
@@ -26,7 +26,7 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// Logging HTTP transport
+// Package auth provides HTTP transport logging
```

Comment:
Oh yeah, not sure what I was thinking. I just need to delete that package comment and add a doc.go or a general comment to google.go